swarm/storage/mru/resource.go

```diff
@@ -916,16 +935,50 @@ func isSafeName(name string) bool {
 	return validname == name
 }
 
+// if first byte is the start of a multihash this function will try to parse it
+// if successful it returns the length of multihash data, 0 otherwise
+func isMultihash(data []byte) int {
```

Comment:
Why not use the multihash package?

Reply:
ok. Fixed in #704 and removed this function